src/pages/publication/PublicationForm.tsx

```diff
@@ -96,12 +96,12 @@ const PublicationForm: FC<PublicationFormProps> = ({ identifier, closeForm }) =>
 
   const validateForm = (values: Publication) => {
     const {
-      reference: { publicationInstance, publicationContext },
+      reference: { publicationContext },
     } = values.entityDescription;
     try {
       validateYupSchema<Publication>(values, publicationValidationSchema, true, {
-        publicationInstanceType: publicationInstance.type,
         publicationContextType: publicationContext.type,
+        publicationStatus: publication?.status,
```

Comment:
vil ikke denne alltid være true? Er publicationStatus noen gang tom?

Reply:
mao: er det nødvendig å sjekke på publicationStatus her?

Reply:
status i seg selv skal vel aldri være tom nei, men publication kan være tom her for første render (har en `useFetchPublication` lenger opp), og det er det vi tar hensyn til

Reply:
ah, skjønner